docs/designers-developers/developers/themes/theme-support.md

```diff
@@ -280,16 +280,18 @@ add_theme_support( 'custom-line-height' );
 
 ### Support custom units
 
-In addition to pixels, users can use other units to define sizes, paddings... The available units are: px, em, rem, vh, vw. Themes can enable support for this feature with the following code:
+In addition to pixels, users can use other units to define sizes, paddings... The available units are: px, em, rem, vh, vw.
+
+Themes can also filter the available custom units.
 
 ```php
-add_theme_support( 'custom-units' );
+add_theme_support( 'custom-units', 'rem', 'em' );
 ```
 
-Themes can also filter the available custom units.
+Custom units are available by default. However, themes can disable support for this feature with the following code:
 
 ```php
-add_theme_support( 'custom-units', 'rem', 'em' );
+add_theme_support( 'custom-units', false );
```

Comment:
I think we should change this to `remove_theme_support( 'custom-units' );` and add a `add_theme_support( true )`  call to Core.

Reply:
Cool! That API makes sense to me. I've never worked with `remove_theme_support` before.. Is there an example I can follow to do that? :D